frontend/Store.js

```diff
@@ -600,6 +604,13 @@ class Store extends EventEmitter {
       clearInterval(requestInt);
       this.capabilities = assign(this.capabilities, capabilities);
       this.emit('connected');
+
+      const traceUpdates = get(SESSION_STORAGE_TRACE_UPDATES_ENABLED_KEY, false);
```

Comment:
Based on the Flow type:
```js
traceupdatesState: ?ControlState;
```
Shouldn't the default value either be `new Record({enabled: false})` or just `null`?

Reply:
@bvaughn not sure if we need to mess up `boolean` data which comes from storage with an instance of `Record`.  So will change to be `null` here.